src/core/MOM_barotropic.F90

```diff
@@ -2063,12 +2112,12 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
         !$OMP do schedule(static)
         do J=jsv-1,jev ; do i=isv-1,iev+1
           v_accel_bt(i,J) = v_accel_bt(i,J) + wt_accel(n) * &
-              ((Cor_v(i,J) + PFv(i,J)) - vbt(i,J)*Rayleigh_v(i,J))
+              ((Cor_v(i,J) + PFv(i,J)) - vbt(i,J)*Rayleigh_v(i,J) - Drag_v(i,J))
```

Comment:
Here and elsewhere , `Drag_u` and `Drag_v` have been added to sums without any parentheses to specify the order of operations.  Because floating point math is not commutative we very strongly encourage sums of 3 or more numbers to include enough parentheses to specific the order of arithmetic, as is discussed at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide. (`(a + b) + c` is does not always give the same value as `a + (b + c)`; for example, `(-1. + 1.) + 1e-20 = 1e-20` but `-1. + (1. + 1e-20) = 0.0` when evaluated using IEEE 64-bit floating point numbers.)

Reply:
I did not realize this but it has been updated now.